### PR TITLE
Mock #get badges endpoint

### PR DESCRIPTION
**Pull request by JohnLackland**
Review my endpoint, please!

### CODE REVIEW

File: ASP.NET-Core-Webapp.UnitTests/HelloServiceTests.cs [hunk 1]
```diff
@@ -23,5 +29,17 @@ public void HelloWorldReturnIsInvalid()
         {
             Assert.NotEqual("Yo Mate!", helloService.HelloWorld());
         }
+
+        [Fact]
```
Comment (molnarln) [inline ASP.NET-Core-Webapp.UnitTests/HelloServiceTests.cs]: Please delete this test case.

**Review by molnarln — COMMENTED** [review]
Added 1 comment in the HelloServiceTest.cs.
All the other staff looks good!

File: ASP.NET-Core-Webapp.UnitTests/HelloServiceTests.cs (hunk 1 above)
Comment (JohnLackland) [reply ASP.NET-Core-Webapp.UnitTests/HelloServiceTests.cs]: Done!

Comment (molnarln) [reply ASP.NET-Core-Webapp.UnitTests/HelloServiceTests.cs]: Thanks!

File: ASP.NET-Core-Webapp.IntegrationTests/Scenarios/MockPostBadgesEndpointTest.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
+﻿using ASP.NET_Core_Webapp.Entities;
```
Comment (szirmi) [inline ASP.NET-Core-Webapp.IntegrationTests/Scenarios/MockPostBadgesEndpointTest.cs]: this file is not part of this PR, remove it

File: ASP.NET-Core-Webapp/Controllers/BadgeController.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                }
+
+                badge.Levels.Add(new LevelEntity() { Description = "New test skill added", Level = 500, Holders = holdersTest });
+
+                return Created("/badges", new { message = "Success" });
+
+            //}
+            //else
+
+            //    return StatusCode(401, new { error = "Unauthorized" });
```
Comment (szirmi) [inline ASP.NET-Core-Webapp/Controllers/BadgeController.cs]: please always remove unnecessary code

**Review by szirmi — CHANGES_REQUESTED** [review]
same as PR2 (mock post EP): UnitTestProject1 and packages shouldn't be in repo

File: ASP.NET-Core-Webapp/Controllers/BadgeController.cs (hunk 3 above)
Comment (JohnLackland) [reply ASP.NET-Core-Webapp/Controllers/BadgeController.cs]: Done!

**Review by szirmi — APPROVED** [review]

**Pull request merged**